include/lsst/meas/base/PixelFlags.h

```diff
@@ -78,9 +78,10 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
 
 private:
     Control _ctrl;
-    KeyMap _centerKeys;
-    KeyMap _centerAllKeys;
-    KeyMap _anyKeys;
+    // The afw Table Keys for the various fields we add to the schema on construction.
+    KeyMap _centerKeys;     // Keys for bits set anywhere within the 3x3 region around the centroid.
+    KeyMap _centerAllKeys;  // Keys for bits set on the entire 3x3 region around the centroid.
+    KeyMap _anyKeys;        // Keys for bits set anywhere in the footprint.
```

Comment:
I think the comment should at least say that the key is the name of the mask plane.

Reply:
Better?

    // The mask name->afw Table Key map for the various fields we add to the schema on construction.